pkg/render/logstorage_test.go

```diff
@@ -267,6 +267,104 @@ var _ = Describe("Elasticsearch rendering tests", func() {
 
 		Context("Deleting LogStorage", deleteLogStorageTests(operatorv1.ClusterManagementTypeStandalone))
 
+		Context("Updating LogStorage resource", func() {
+			It("should create new NodeSet", func() {
+				logStorage2 := &operator.LogStorage{
```

Comment:
Minor: Why is this called `logStroage2`? The number 2 seems like noise (not important).